packages/service-registry-api/src/primitives.rs

```diff
@@ -62,3 +62,21 @@ pub enum AuthorizationState {
     Authorized,
     Jailed,
 }
+
+// Represents any modifiable fields of the Service struct
+// Any non-None field overwrites the value currently stored in the Service object
+#[cw_serde]
+pub struct UpdatedServiceParams {
+    pub min_num_verifiers: Option<u16>,
+    pub max_num_verifiers: Option<Option<u16>>,
+    pub min_verifier_bond: Option<nonempty::Uint128>,
+    pub unbonding_period_days: Option<u16>,
+}
+
+// Represents any overrideable fields of the Service struct
+// Any non-None field overrides the value currently stored in the Service object
+#[cw_serde]
+pub struct ServiceParamsOverride {
+    pub min_num_verifiers: Option<u16>,
+    pub max_num_verifiers: Option<Option<u16>>,
```

Comment:
why double wrapped option?

Reply:
if I remember correctly, one of the reasons why we kept it that way is because we already have a precedence with `UpdatedServiceParams`, I think consistency is better in this case unless we are ok introducing a breaking change. Is it really a breaking change considering we don't use `max_num_verifiers` yet and it's only callable by governance?